Makefile.am

```diff
@@ -63,7 +63,7 @@ all-local: .stamp-distutils-in-builddir config.py cupshelpers/config.py
 
 # Use distutils to install the module.
 install-exec-local: .stamp-distutils-in-builddir
-	$(PYTHON) setup.py install --prefix=$(DESTDIR)$(prefix)
+	$(PYTHON) setup.py install --root=$(DESTDIR) --prefix=$(DESTDIR)$(prefix)
```

Comment:
Hi Rafael,

thank you for the PR!

Actually the similar fix was brought into Fedora when Python 3.12 was introduced and it works fine since that:

```
--- a/Makefile.am
+++ b/Makefile.am
@@ -63,7 +63,7 @@ all-local: .stamp-distutils-in-builddir config.py cupshelpers/config.py
 
 # Use distutils to install the module.
 install-exec-local: .stamp-distutils-in-builddir
-       $(PYTHON) setup.py install --prefix=$(DESTDIR)$(prefix)
+       $(PYTHON) setup.py install --root $(DESTDIR) --prefix=$(prefix)
 
 # Uninstall the module, crossing our fingers that we know enough
 # about how distutils works to do this.  Unfortunately, distutils
```

I see the same patch was used in other project - https://github.com/AOSC-Dev/aosc-os-abbs/commit/7b298dd999eaddc1305913dadf47f0f677a5b567 - so I will go with this fix.

But either way thank you for PR! I'll merge it and then update it to match with Fedora changes.